docs/howto/openstack/nova/move-instance-between-places.md

```diff
@@ -0,0 +1,107 @@
+# Move an instance from one Cleura Cloud region to another
+
+This guide will show you how to move or copy your instance to a different region in Cleura cloud.
+
+## Prerequisites
+
+- Installed OpenStack [CLI client](https://docs.cleura.cloud/howto/getting-started/enable-openstack-cli/)
+- Access to source and target location (RC-file)
+- Enough space on the local machine that you download your server to.
+- Target location configured with Network, Security Groups etc
+
+>To speed up this work, we recommend to use a temporary linux instance located at the same region as the source. This make the transfer faster not needing to downloading to and uploading from your local machine.
+
+## Recommendations
+
+First step will be to create a snapshot of the instance, during this step it is recommended to shutdown the instance to ensure data consistency. Once the snapshot is done the instance can be rebooted again.
+
+Use the UUID of the instance instead of using the instance name, this will make sure you are using the correct instance.
+
+To get the correct UUID of your instance you can use the following command:
+
+```bash
+openstack server list
+```
+
+## Creating a snapshot
+
+To start the snapshot process you use the following command:
+
+```bash
+openstack server image create --wait --name <new-image-name> <uuid-of-instance>
+```
+
+After a while you will get some output showing information of the new image, such as the current state and the image UUID.
+
+Before going on the state needs to be active and you can see the current state with the following command:
+
+```bash
+openstack image show <uuid>
+```
+
+## Download your snapshot
+
+Before you can download the image you need to know what format the image has, to find this out you can run the following command:
+
+```bash
+openstack image show <image_id> | grep disk_format
+```
+
+To download the image to the local machine you use the following command:
+
+```bash
+openstack image save --file <local image name>.<disk_format> <your_image_id>
+```
+
+When the file have been downloaded, you are done with the steps on the source side, and the following steps will be done on the target side.
```

Comment:
"The file" is singular, so this should say "has been downloaded". But again, active voice is better than passive, so you can write "When you have downloaded the file" instead.

Reply:
Also, don't use "on the [source|target] side" here. Rather, say "for the source region" and "for the target region".